### ISSUE TITLE
Encountering Large Data Out in Azure when Tail is Enabled

### ISSUE THREAD

**Issue by Mike-E-angelo** [datalust/seq-tickets#1439]
**Describe the bug**
When Tail is enabled in the Seq server user interface, Data Out usage in the Azure App Service reports between 4-5MB per second of usage.  This seems abnormally high.

**To Reproduce**
Install Seq for Docker as explained here: https://ml-software.ch/posts/running-seq-on-azure

Application Settings:
```
ACCEPT_EULA = Y
SEQ_API_CANONICALURI = https://<server-uri>
SEQ_FEATURES_ENABLED = nativemetastore
```

Path Mappings:
- Premium File Share

**Expected behavior**
Not as much usage. :)  In this case, I found out about it when my instance stopped due to hitting a quota.  Thankfully I am in my first month with the $200 free credits so I can leverage this.

**Screenshots**
Sample usage:
![](https://i.imgur.com/Gdfx1zt.png)

**Environment (please complete the following information):**
 - OS: `datalust/seq:latest`
 - Browser Chrome
 - Seq Version `2021.4.6986`
 - Using Docker? Yes

**Additional context**
Note that the network usage for Chrome as denoted by Task Manager is an expected .1Mb/s.

**Comment by nblumhardt** [comment]
Thanks for the note, @Mike-E-angelo.

The chart seems to show more like 2-4 MB/minute to me, if you take the sum of 35.4 MB, and divide that by the duration of the chart, or by the duration covered by the peaks. Is that me misreading the metrics?

**Comment by Mike-E-angelo** [comment]
Ah!  Apologies, @nblumhardt.  Those spikes are when I had Tail enabled.  Perhaps a better screenshot would have been when my quota was met after 1 hour of having Tail enabled and it was a consistent line around the 5MB level. :)

So to be sure here, the provided screenshot is from enabling and disabling Tail a few times to demonstrate that it is the culprit.  Please let me know if that makes sense and/or if there is any further information you require from me to further diagnose.

**Comment by nblumhardt** [comment]
Thanks, Mike ! Those numbers are per minute, though, if I'm reading the chart correctly? (Your ticket states 4-5MB/sec.)

Definitely still room to improve this, but the numbers are in line with what we'd expect, for the polling mechanism that tailing uses currently.

**Comment by Mike-E-angelo** [comment]
> Your ticket states 4-5MB/sec.

It sure does because I am a total newb, @nblumhardt 😁😇  Well, what got me to report this is that I ended up meeting a quota within an hour of usage.  I had to scale out my instance, so that was the concern and the driver behind reporting this.

If I am hearing correctly, this is a known issue, but operating as expected.  That is OK with me then, as well. :)

Thank you again for all your great work out there, and for your patience with me as I learn the ropes here. 🙏
